engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingformula/BooleanRollingFormulaOperator.java

```diff
@@ -64,11 +63,12 @@ protected Context(final int affectedChunkSize, final int influencerChunkSize) {
             final SingleValueColumnSource<ObjectVector<?>> formulaInputSource =
                     (SingleValueColumnSource<ObjectVector<?>>) SingleValueColumnSource
                             .getSingleValueColumnSource(inputVectorType);
-            formulaInputSource.set(new ObjectRingBufferVectorWrapper(windowValues, inputVectorType));
+            // noinspection rawtypes
+            formulaInputSource.set(new ObjectRingBufferVectorWrapper(windowValues, inputComponentType));
```

Comment:
`I didn't fully understand this change, can you please explain?`

Fixing an earlier bug.  I'm trying to track the fundamental component type of the input Object column to the Vector created from it.

`Also, can we use ByteRingBufferVectorWrapper here since we would have byte values internally? Please correct me if I am mistaken or if this is out of scope for this PR.`

This matches AggFormula in creating ObjectVector<Boolean> when assembling groups of Boolean input. The functions in `Basic.java` expect this input.